ios/Classes/UADeepLinkAction.m

```diff
@@ -0,0 +1,30 @@
+/* Copyright 2017 Urban Airship and Contributors */
+
+#import "UADeepLinkAction.h"
```

Comment:
Can we rename this to something else so it wont conflict with our SDK when we introduce a class called UADeepLinkAction? Maybe UATiDeepLinkAction?